### PR TITLE
Remove completePayment from BookingSummaryController

### PR DESCRIPTION
**Pull request by joelchoo**

### CODE REVIEW

File: beeline/services/paymentService.js [hunk 1]
```diff
@@ -31,9 +30,6 @@ angular.module("beeline").factory("PaymentService", [
     // book is booking Object
     async function completePayment(paymentOptions, book) {
```
Comment (shen-xujing) [inline beeline/services/paymentService.js]: this method is triggered in bookingSummaryModalService, need to move the state transition and error handling part to there also

Comment (joelchoo) [reply beeline/services/paymentService.js]: Hmm I don't see it in `bookingSummaryModalService`? I only see `PaymentService.payHandler`

File: beeline/services/paymentService.js [hunk 2]
```diff
@@ -48,24 +45,35 @@ angular.module("beeline").factory("PaymentService", [
...
+        TicketService.setShouldRefreshTickets()
+      } finally {
+        RoutesService.fetchRoutePasses(true)
+        RoutesService.fetchRoutePassCount()
+        RoutesService.fetchRoutesWithRoutePass()
 
-        $ionicLoading.hide()
+        CreditsService.fetchReferralCredits(true)
+        CreditsService.fetchUserCredits(true)
+      }
```
Comment (shen-xujing) [inline beeline/services/paymentService.js]: line 54 and 55 can be removed

**Comment by joelchoo** [comment]
Tested and seems to work fine

**Pull request merged**